### ISSUE TITLE
Contours03.dxf as background pic

### ISSUE THREAD

**Issue by OakCastle** [OpenOrienteering/mapper#1174]
### Steps to reproduce (can't be reproduced on two of out three computers)
1. Georef your map (EPSG #3006)
2. Open a background file (Contouts03.dxf)
3. You don't see the "Use same georef as the map" box

### Actual behaviour
Just back from a meeting with two other mappers (OOM newbees) on how to use OOM on PC and Tablet. During our session one guy experienced problems adding the Kartapullautin file Contours03.dxf to the test mapping project (as a background file). Instead of seeing the normal dialouge box with "Use the map coordinate system" (or something similar) he was met by the dialogue box that appears if the  map has not been geo-referenced/for line objects. The thin purple lines you normally see in the Countours03 file was also very thick and non usable in combination with pictures of inclination to get that great 3D feel you normally see.       

### Expected behaviour
Two of us managed to open the same Contours03 file as a georeferenced background pic, and got the question if to use the same georef.sys as the map project. Our 30 cm contour lined were all nice and thin compared to the other guys fat purple lines. All three computers have the 0.8.3 mapper version installed. We georeferenced all maps exactly the same way using the SWEREF 99 system (EPSG code 3006).

A very strange problem as two of us saw no problems while the third one did. Not sure if the problem has something to do with his computer. Is there someone out there that has seen something similar, and know of any solution to the problem.

The contours looks very similar to the ones you see if you use the Contours03 file as a background on a Android tablet (fat, ugly and non usable lines). This is another problem that you can manage by a workaround (by importing the Contours03 as map objects and change its symbol to a 0,01 mm thin purple line) - is there a better way to handle this problem on an Android...as the file turns very heavy when the contours are imported?     

### Configuration
Mapper Version: 0.8.3
Operating System: Windows 10

**Comment by dg0yt** [comment]
You are probably affected by #1166 (which I discovered in a newbie workshop, too). The desired behaviors is achieved either with DXF disabled or enabled in the GDAL/OGR settings.

Note that "thin" and "fat" line doesn't mean much here: GDAL/OGR uses a line which scales with zoom (internally defined by a line symbol), while the legacy importer always draws one-pixel lines.

**Comment by Symbian9** [comment]
> "Contours03.dxf as background pic"

Issue title should be renamed (at-least) to "DXF as template".

In Mapper there NO term "background pic", instead use term "template" for it.

**Comment by OakCastle** [comment]
Ok, I use the swedish version and therefor aren’t used to the words used in
the english version👍🏻

Fredrik


mån 12 nov. 2018 kl. 20:34 skrev app4soft <notifications@github.com>:

**Comment by dg0yt** [comment]
> Ok, I use the swedish version and therefor aren’t used to the words used in the english version.

No problem for me.

I understand that a) you miss the "Use same georef as the map" dialog step, and b) you experience fat, "ugly and non usable lines". But really, you need to tell me the desired appearance what by DXF *enabled* or *disabled* in the GDAL/OGR settings, not by thin or fat. I'm about to remove the *disabled* option in v0.9 (#1160). And if there is a release v0.8.4, it will (now correctly) default to the *enabled* option.

**Comment by OakCastle** [comment]
Ok, great!

What I meant by fat is that on the Android application, and now on 1 our of
3 computers (2 were OK), the normally thin (1 pixel or so) template with
the 30 cm contour line is about 5-7 pixels wide and therefor too thick use
as a tool for enganced depth perception. As for 2 of us we did not
experience this problem om our PC's with the 1. wider contour lines on the
template as well as 2. the templates' inability to become georeferenced -
only 1 persons PC did show this behaviour. On the Android application
though the 1 pixel or so contour line of the countours03.dxf file becomes
5-7 pixels - and therefor too wide to use - has forced me apply a work
around that include import of the 30 cm contour line followed by changing
its symbol to a selfmade one with width 0,01 cm. If this probelm could be
fixed on the Android application I would be highly grateful due to the faxt
that the file becomed very heavy when having a 50-100 MB contour lines as
editable symbols. When working with the map file/project on my PC I turn
the 30 cm contours template on and the 0,01 cm contour symbol off - and
when using the Android application in field it's the other way round.

In short:
1. One (out of three) person experienced problem with the Contours03.dxf
template - I was not affected, but would like to help him get back on track.
2. In the Android application the normally (on PC) 1 pixel thin 30 cm
countour lines becomes 5-7 pixels wide which force me to apply a not
perfect workaround to thin the lines to a managebley width..

I hope I made it clear enough...

Best regards,
Fredrik

Den tis 13 nov. 2018 kl 09:02 skrev Kai Pastor <notifications@github.com>:

**Comment by dg0yt** [comment]
Again, you need to tell me if the desired appearance is achieved with DXF *enabled* or *disabled* in the GDAL/OGR settings.
And you may need to change the relevant setting at least once before you can rely on it being effective, due the known issue  #1166 I linked in my first answer.

**Comment by OakCastle** [comment]
I don't know (yet) as it's not on my computer, but will ask the person
affected and return his ansver to you.

/Fredrik

Den tis 13 nov. 2018 kl 09:44 skrev Kai Pastor <notifications@github.com>:

**Comment by dg0yt** [comment]
Without more information, this cannot be handled. Please reopen when you have more information.
